stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -47,9 +47,12 @@ public type OutboundOAuth2Provider object {
         };
     }
 
-    # Generate token for OAuth2 authentication.
-    #
-    # + return - Generated token or `auth:Error` if an error occurred
+# Generate token for OAuth2 authentication.
+# ```ballerina
+# string:auth:Error token = outboundOAuth2Provider.generateToken();
+# ```
+#
+# + return - Generated `string` token or `auth:Error` if an error occurred
```

Comment:
```suggestion
# + return - Generated `string` token or `auth:Error` if an error occurred
```
Please refer the template for the correct format of the return statement description.

Reply:
```suggestion
# + return - Generated `string` token or else `auth:Error` if an error occurred
```